### PR TITLE
[HNC-515] - Add Frogbot scans to Pentaho actions-common PRs

### PR DESCRIPTION
**Pull request by AdityaHitachi**

### CODE REVIEW

**Review by vganpisetty — APPROVED** [review]

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -143,6 +143,13 @@ jobs:
           sonar_utility: sonar-scanner
           sonar_commands: '("-Dsonar.projectKey=${{env.SONAR_PROJECT_KEY}} -Dsonar.host.url=${{env.SONAR_HOST_URL}} -Dsonar.login=${{env.SONAR_LOGIN}} -Dsonar.pullrequest.key=${{github.event.number}} -Dsonar.pullrequest.branch=${{github.event.pull_request.head.ref}} -Dsonar.pullrequest.base=${{github.event.pull_request.base.ref}}")'
 
+      - name: FrogBot
+        uses: lumada-common-services/gh-composite-actions@stable
+        env:
+          JF_URL: https://hitachi.jfrog.io/
```
Comment (cardosov) [inline .github/workflows/pr.yml:149]: This should be `https://one.hitachivantara.com`

File: .github/workflows/pr.yml [hunk 2]
```diff
@@ -143,6 +143,13 @@ jobs:
...
+          JF_ACCESS_TOKEN: ${{ secrets.JF_ACCESS_TOKEN }}
+          JF_GIT_TOKEN: ${{ github.token }}
```
Comment (cardosov) [inline .github/workflows/pr.yml:151]: Why do we need `JF_GIT_TOKEN` again? I don't remember... is it to allow Jfrog to comment on the PR maybe?!

**Review by cardosov — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 2 above)
Comment (AdityaHitachi) [reply .github/workflows/pr.yml:151]: There is no specific mention from FrogBot regarding why this is required, but according to their [documentation,](https://github.com/jfrog/frogbot/blob/821a26f50338a319d238f9bbc054f6b7a66336ad/.github/workflows/frogbot-scan-pull-request.yml#L26) it is mandatory.

File: .github/workflows/pr.yml (hunk 1 above)
Comment (AdityaHitachi) [reply .github/workflows/pr.yml:149]: "I haven't tested FrogBot with this URL. I will test and update.

Comment (AdityaHitachi) [reply .github/workflows/pr.yml]: updated the URL

**Review by cardosov — APPROVED** [review]

**Pull request merged**